public/student.html

```diff
@@ -194,12 +198,12 @@ <h1 id="danny">Danny Fraser</h1>
       </div>
       <div class="col-lg-6   pl-lg-5 pl-1 pt-lg-0 pt-2 py-5 d-flex flex-column justify-content-lg-around">
          <div class>
-            <h1 class="hyphenate" id="tasha">Natasha Griffin</h1>
+            <h3 class="h1 hyphenate" id="tasha">Natasha Griffin</h3>
             <p class="lead border-dark border-bottom">Oregon State University, College of Earth, Ocean, and Atmospheric Sciences</p>
             <p>I&apos;m working with Dr. Byron Crump to better understand what microbes do in the Beaufort lagoons. My thesis focuses on tracking microbial activity across seasons in lagoon waters and sediments, as well as exploring how microbial processes relate to biogeochemical rates like nitrogen cycling and carbon fixation in the lagoons. My broader interests include ecosystem ecology, biogeochemistry, and science communication. When I&apos;m not doing science, I love spending time on the Oregon coast or visiting the desert in my home state of Utah. </p>
             <button class="btn btn-outline-primary m-1"><a href="mailto:grifnata@oregonstate.edu"><i class="  far fa-paper-plane"></i>  grifnata@oregonstate.edu</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://scholar.google.com/citations?user=0c7X7aMAAAAJ&amp;hl=en" target="_blank">Publications</a></button>
-            <button class="btn btn-outline-primary m-1"><a href="https://natashaagriffin.wordpress.com/" target="_blank">Personal website</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://scholar.google.com/citations?user=0c7X7aMAAAAJ&amp;hl=en" target="_blank" aria-label="open page in new tab">Publications</a></button>
+            <button class="btn btn-outline-primary m-1"><a href="https://natashaagriffin.wordpress.com/" target="_blank" aria-label="open page in new tab">Personal website</a></button>
```

Comment:
specify what you're opening in new tab

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5